src/backend/base/langflow/components/brightdata/brightdata_web_scraper.py

```
@@ -0,0 +1,189 @@
+                    "status": "success",
+                    "content_length": len(content),
+                    "zone": self.zone_name,
+                    "response_headers": dict(response.headers),
+                    "status_code": response.status_code
+                }
+                
+                return Data(
+                    text=content,
+                    data=self._metadata
+                )
+            else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)